youtube_dl/extractor/cba.py

```diff
@@ -0,0 +1,104 @@
+        }
+    }
+    _FORMATS = {
+        'audio/ogg': {'id': '1', 'ext': 'ogg', 'preference': 100},
+        'audio/mpeg': {'id': '2', 'ext': 'mp3', 'preference': 50}
+    }
+    _API_KEY = None
+
+    def __init__(self, *args, **kwargs):
+        try:
+            self._API_KEY = os.environ["CBA_API_KEY"]
```

Comment:
youtube-dl is wide targeted. Absence of any authentication is preferred since not everybody may have/want to bother to obtain an API key.